package.json

```diff
@@ -172,7 +166,9 @@
       "typescript"
     ]
   },
-  "dependencies": {},
+  "dependencies": {
+    "expo-dev-client": "~5.2.1"
```

Comment:
Correct me if I'm wrong, but I wouldn't expect this package at all in the published package.